varnishgather

```diff
@@ -608,8 +608,14 @@ mycat /proc/sys/net/ipv4/tcp_tw_reuse
 for pid in ${PID_VARNISHD}; do
 	run cat -v /proc/$pid/cmdline
 	mycat /proc/$pid/cgroup
+	show_limits $pid
+done
+
+for file in /etc/security/limits.conf /etc/security/limits.d/*.conf
+do
+	test -f $file || continue
```

Comment:
The cat (`mycat`)  does a "-r", isnt that good enough?

(Also, for some reason it incrememts task-"item-id", dunno why :))